### PR TITLE
Update sbt to 1.4.1

### PR DESCRIPTION
**Pull request by tgodzik**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -292,6 +292,8 @@ val mtagsSettings = List(
     baseDirectory.in(ThisBuild).value / "mtags",
     scalaVersion.value
   ),
+  // @note needed to deal with issues in https://github.com/scalameta/metals/pull/2157
+  sources in (Compile, doc) := Seq.empty,
```
Comment (tgodzik) [inline build.sbt:296]: Since Metals is not a library, I imagine this will not be needed.

**Comment by tgodzik** [comment]
Follow up from https://github.com/scalameta/metals/pull/2157

File: build.sbt (hunk 1 above)
Comment (ckipp01) [reply build.sbt:296]: Ahh, good point. I should have thought of this before. 👍

**Review by ckipp01 — APPROVED** [review]
LGTM!

**Pull request merged**